modules/styling/stories/mdx/Utilities.mdx

```diff
@@ -0,0 +1,279 @@
+import {ExampleCodeBlock, SymbolDoc} from '@workday/canvas-kit-docs';
+import {CreateStyles} from './examples/CreateStyles';
+import {CreateVars} from './examples/CreateVars';
+
+<Meta title="Styling/Utilities" />
+
+# Canvas Kit Styling Utilities
+
+## cs Prop
+
+The `cs` prop takes in a single, or an array of values that are created by the `cs` function, a
+string representing a CSS class name, or the return of the `createVars` function. It merges
+everything together and applies `className` and `style` attributes to a React element. Most of our components extend the `cs` prop so that you can statically apply styles to them.
+
+>**Important**: While the `cs` prop accepts a style object, **this will not** be considered statically styling an element and you will lose the performance benefits. By default, our components extend the `cs` prop.
+
+```tsx
+import {system} from '@workday/canvas-tokens-webs'
+import {PrimaryButton} from '@workday/canvas-kit-react/button'
+
+const styles = createStyles({ color: system.color.static.red.default });
+
+function MyComponent() {
+  return <PrimaryButton cs={styles}>Text</PrimaryButton>;
+}
+```
+
+<SymbolDoc headingStart={2} name="handleCsProp" />
+
+<SymbolDoc headingStart={2} name="mergeStyles" />
+
+>**Note:** While we support `mergeStyles` we'd advise against using this in your components so that users can get the performance benefit of static styling using utilities like `createStyles` and `createStencil` in tandem with the `cs` prop.
```

Comment:
Now that this doc has been renamed to `Utilities`, I feel like this content for `handleCSProp` and `mergeStyles` can be moved to the Merging Styles page.

```suggestion
```